City-of-Helsinki/outdoors-sports-map
# Update service info text

## Description
  Update service info: 
   - updates service info with Status color codes info


## Context

[HULK-20](https://project.anders.fi/browse/HULK-20)

## How Has This Been Tested?

Click on info icon and see if `Information on the service` includes the `Status color codes` info

## Manual Testing Instructions for Reviewers

Click on info icon and see if `Information on the service` includes the `Status color codes` info

## Screenshots

![updated-information-on-the-service](https://user-images.githubusercontent.com/117433931/225870109-d7ecb474-e47f-41fc-b24e-11706ee16078.png)
